### PR TITLE
feat(Text): add TinyText component

### LINKED ISSUE #1377: Add a TinyText component
## Feature description
In designs (on Desktop) we want to have the option for a smaller text size than the SmallText, for example in dialog subtitles:
Screenshot 2022-02-22 at 11 57 12

At the moment there are already quite some places where we use this text size, but it's not an official component yet.

The text size would be `0.875rem` (14px) with a default line height of `1.5` as with all other text styles. On mobile the text size shouldn't change (14px is as low as we want to go).

### PR DESCRIPTION
**Pull request by Emacdyz**
# Description

Creation of the new TinyText 

## Changes

- [x] Exported new TinyText component, following design guidelines 
- [ ] Added related story

## How to test

- Checkout this branch
- `$ yarn run storybook`
- Go to TinyText story and check that everything is ok

## To be notified

@Marruk 

## Links
fixes https://github.com/TicketSwap/solar/issues/1377

### CODE REVIEW

**Comment by bobbonius** [comment]
You need to export it here in order for it to be useable in the build:

https://github.com/TicketSwap/solar/blob/a8eefd51047a8bff0c69b0ae1e27319687196dde/src/components/index.ts#L11

**Review by bobbonius — APPROVED** [review]
😎 🔍

File: src/components/Text/index.tsx [hunk 1]
```diff
@@ -37,3 +37,13 @@ export const SmallText = styled.span`
...
   }
 `
+
+export const TinyText = styled.span`
+  ${baseTextStyles};
+  font-size: ${fontSize[14]};
+  color: ${color.spaceLight};
+
+  @media ${device.tablet} {
+    font-size: ${fontSize[14]};
```
Comment (diogoredin) [inline src/components/Text/index.tsx:47]: Is this necessary if it's the same value as the other?

**Review by diogoredin — APPROVED** [review]
Left you a question, but LGTM

File: src/components/Text/index.tsx (hunk 1 above)
Comment (Emacdyz) [reply src/components/Text/index.tsx:47]: Yes it overrides the fontsize[18] that is coming from `${baseTextStyles}`

Comment (diogoredin) [reply src/components/Text/index.tsx:47]: But we have font-size: `${fontSize[14]};` right after `${baseTextStyles};`, no? 🤔

Comment (diogoredin) [reply src/components/Text/index.tsx:47]: This was a overwrite from the parent, got it 😅

**Pull request merged**